packages/twenty-front/src/modules/object-record/record-field/meta-types/display/components/NumberFieldDisplay.tsx

```diff
@@ -6,11 +6,15 @@ export const NumberFieldDisplay = () => {
   const { fieldValue, fieldDefinition } = useNumberFieldDisplay();
   const decimals = fieldDefinition.metadata.settings?.decimals;
   const type = fieldDefinition.metadata.settings?.type;
-  const value =
-    type === 'percentage' && fieldValue
-      ? `${formatNumber(Number(fieldValue) * 100, decimals)}%`
-      : fieldValue
-        ? formatNumber(Number(fieldValue), decimals)
-        : null;
+  
+  if (!fieldValue) return <NumberDisplay value={null} />;
```

Comment:
Nice one. Faster and consistent with our way of returning instead of handling else statement